src/main/java/com/google/api/codegen/transformer/InitCodeTransformer.java

```diff
@@ -81,18 +82,26 @@ public InitCodeView generateRequestObjectInitCode(MethodTransformerContext conte
     List<GapicSurfaceTestAssertView> assertViews = new ArrayList<>();
     Map<String, Object> initFieldStructure = createInitFieldStructure(context);
     InitCodeGenerator generator = new InitCodeGenerator();
+
     InitCode initCode =
         generator.generateRequestFieldInitCode(context.getMethod(), initFieldStructure, fields);
 
+    ArrayList<String> fieldNames = new ArrayList<>();
+    for (Field field : fields) {
+      fieldNames.add(Name.from(field.getSimpleName()).toLowerCamel());
+    }
```

Comment:
1. Don't use casing functions in transformers
2. initCode.getArgFields() should give you what you need.